src/metrics/pcloud.jl

```diff
@@ -0,0 +1,39 @@
+export chamfer_distance
+
+#TODO: change PointCloud to 3xV format
+# chamfer_distance(A::PointCloud, B::PointCloud; w1::Number=1.0, w2::Number=1.0) =
+#     _chamfer_distance(A.points, B.points, Float32(w1), Float32(w2))
+
+chamfer_distance(A::AbstractArray{<:Number}, B::AbstractArray{<:Number}; w1::Number=1.0, w2::Number=1.0) =
```

Comment:
Personally, I don't like the user giving a Float64 array and getting a Float32 output. I would suggest, having a normal function for only float32. For the more general case, show a simple warning that the inputs are being typecasted

Reply:
Should I restrict vertices to be Float32? I have kept verts as AbtractFloat.